charts/moco/README.md

```diff
@@ -58,6 +58,104 @@ Users must manually upgrade the CRD if there is a change in the CRD used by MOCO
 
 https://helm.sh/docs/chart_best_practices/custom_resource_definitions/#install-a-crd-declaration-before-using-the-resource
 
+## Migrate to v1
+
+Chart version v1 has breaking changes.
+The `.metadata.name` of the resource generated by Chart is changed.
+
+e.g.
+
+* `{{ template "moco.fullname" . }}-foo-resources` -> `moco-foo-resources`
+
+Related Issue: [cybozu-go/moco#426](https://github.com/cybozu-go/moco/issues/426)
+
+If you are using a release name other than `moco`, you need to migrate.
+
+The migration steps involves deleting and recreating each MOCO resource once, except CRDs.
+Since the CRDs is not deleted, the pods running existing MySQL clusters are not deleted, so there is no downtime.
```

Comment:
```suggestion
The migration steps involve deleting and recreating each MOCO resource once, except CRDs.
Since the CRDs are not deleted, the pods running existing MySQL clusters are not deleted, so there is no downtime.
```